torch/_inductor/codegen/triton.py

```diff
@@ -74,7 +74,7 @@ def is_aligned(x):
         if isinstance(x, TensorArg):
             return x.buffer not in V.graph.unaligned_buffers
         if isinstance(x, SizeArg):
-            return V.graph.sizevars.maybe_guard_multiple_of(x.expr, ALIGNMENT)
+            return V.graph.sizevars.statically_known_multiple_of(x.expr, ALIGNMENT)
         raise NotImplementedError(f"unhandled {type(x)}: {x}")
 
     if config.triton.divisible_by_16 and not dynamo_config.dynamic_shapes:
```

Comment:
Are you going to drop these `dynamic_shapes` tests on your own later?

Reply:
Yes, that is separate.